src/DynamoCoreWpf/DynamoCoreWpf.csproj

```diff
@@ -161,6 +160,10 @@
     <None Remove="UI\Images\NodeStates\frozen-64px.png" />
     <None Remove="UI\Images\NodeStates\hidden-64px.png" />
     <None Remove="UI\Images\NodeStates\info-64px.png" />
+    <None Remove="UI\Images\NodeStates\package-64px.png" />
+    <None Remove="UI\Images\NodeStates\preview-64px.png" />
```

Comment:
should we rename these to transient as well ?

Reply:
oh yeah, I need to update this as well

Reply:
Updated